src/applications/vaos/tests/e2e/vaos-cypress-helpers.js

```diff
@@ -0,0 +1,178 @@
+  });
+  return data;
+}
+
+function updateTimeslots(data) {
+  const startDateTime = moment()
+    .add(4, 'days')
+    .day(9)
+    .format('YYYY-MM-DDTHH:mm:ss[+00:00]');
+  const endDateTime = moment()
+    .add(4, 'days')
+    .day(9)
+    .add(60, 'minutes')
+    .format('YYYY-MM-DDTHH:mm:ss[+00:00]');
+
+  const newSlot = {
+    bookingStatus: '1',
+    remainingAllowedOverBookings: '3',
+    availability: true,
+    startDateTime,
+    endDateTime,
+  };
+
+  data.data[0].attributes.appointmentTimeSlot = [newSlot];
+
+  return data;
+}
+
+export function initAppointmentListMock() {
+  cy.server();
+  cy.login();
+  cy.route({
+    method: 'GET',
+    url: '**/v0/feature_toggles*',
+    status: 200,
+    response: {
+      data: {
+        features: [
+          {
+            name: 'vaOnlineScheduling',
+            value: true,
+          },
+          {
+            name: 'vaOnlineSchedulingCancel',
+            value: true,
+          },
+          {
+            name: 'vaOnlineSchedulingRequests',
+            value: true,
+          },
+          {
+            name: 'vaOnlineSchedulingCommunityCare',
+            value: true,
+          },
+          {
+            name: 'vaOnlineSchedulingDirect',
+            value: true,
+          },
+          {
+            name: 'vaOnlineSchedulingPast',
+            value: true,
+          },
+        ],
+      },
+    },
+  });
+
+  cy.route({
+    method: 'GET',
+    url: '**/vaos/v0/community_care/supported_sites',
+    response: supportedSites,
+  });
+
+  cy.route({
+    method: 'GET',
+    url: '**/vaos/v0/appointment_requests**',
+    response: updateRequestDates(requests),
+  });
+
+  cy.route({
+    method: 'GET',
+    url: /.*\/v0\/appointments.*type=va$/,
+    response: updateConfirmedVADates(confirmedVA),
+  });
+
+  cy.route({
+    method: 'GET',
+    url: /.*\/v0\/appointments.*type=cc$/,
+    response: updateConfirmedCCDates(confirmedCC),
+  });
+
+  cy.route({
+    method: 'GET',
+    url: '**/vaos/v0/facilities/983/cancel_reasons',
+    response: cancelReasons,
+  });
+
+  cy.route({
+    method: 'PUT',
+    url: '**/vaos/v0/appointments/cancel',
+    response: '',
+  });
+
+  cy.route({
+    method: 'GET',
+    url:
+      '**/vaos/v0/appointment_requests/8a48912a6cab0202016cb4fcaa8b0038/messages',
+    response: {
+      data: [
+        {
+          id: '8a48912a6cab0202016cb4fcaa8b0038',
+          type: 'messages',
+          attributes: {
+            surrogateIdentifier: {},
+            messageText: 'Request 2 Message 1 Text',
+            messageDateTime: '11/11/2019 12:26:13',
+            senderId: '1012845331V153043',
+            appointmentRequestId: '8a48912a6cab0202016cb4fcaa8b0038',
+            date: '2019-11-11T12:26:13.931+0000',
+            assigningAuthority: 'ICN',
+            systemId: 'var',
+          },
+        },
+      ],
+    },
+  });
+
+  cy.visit('health-care/schedule-view-va-appointments/appointments/');
```

Comment:
Seems odd to have these two lines inside a function called `initAppointmentListMock`. Maybe move them to the main test file?

Reply:
Moved these back into the test file.